.codeclimate.yml

```diff
@@ -0,0 +1,36 @@
+---
+engines:
+  duplication:
+    enabled: true
+    config:
+      languages:
+      - ruby
```

Comment:
>  I wonder why this was added.

Because our init code isn't that smart 😀. We see some regexes for some languages duplication supports, so we enable the engine with a default config, but don't tailor that config to *which* regexes matched.